package.json

```diff
@@ -37,7 +37,7 @@
     "colors": "^1.1.2",
     "elasticsearch": "^13.3.1",
     "joi": "^12.0.0",
-    "mergeable": "latest",
+    "lodash.merge": "latest",
```

Comment:
Can you change this to whatever the latest version is? That way Greenkeeper will help us detect breaking changes.

Reply:
Sure, I'll update the PR to reflect that.